page_iterator.go

```diff
@@ -200,7 +200,7 @@ func convertToPage(response interface{}) (PageResult, error) {
 	}
 	value = reflect.NewAt(value.Type(), unsafe.Pointer(value.UnsafeAddr())).Elem()
 
-	nextLink := ref.FieldByName("nextLink")
+	nextLink := ref.FieldByName("odataNextLink")
```

Comment:
I think we should introduce an intermediate interface instead of relying on reflection since Go is duck typing when writing assertions. It'll probably run faster

Reply:
Resolved with https://github.com/microsoftgraph/msgraph-sdk-go-core/pull/105/commits/ea3ab3217b394dd92770be355bef6b6f71faf8e3